testhelpers/org.eclipse.n4js.cli.tests.helper/src/org/eclipse/n4js/cli/helper/N4jsLibsAccess.java

```diff
@@ -368,8 +368,8 @@ private static void assertN4jsLibsAreBuilt(Path n4jsLibsLocation) {
 				"******************************************************************\n" +
 				"Maybe you forgot to run MWE2 workflow " + BuildN4jsLibs.class.getSimpleName() + "?\n" +
 				"******************************************************************";
-		Assert.assertTrue("n4js-runtime does not exist in node_modules folder: " + n4jsRuntimeLink + warning,
-				Files.exists(n4jsRuntimeLink));
+		Assert.assertTrue("not a symbolic link pointing to package \"" + N4JSGlobals.N4JS_RUNTIME + "\": "
+				+ n4jsRuntimeLink + warning, Files.isSymbolicLink(n4jsRuntimeLink));
```

Comment:
Are you here accidentally reverting #1490? (maybe merge mishap)

Reply:
Good catch! Thanks, I'll undo the revert.